wallet-client/CLI.hs

```diff
@@ -24,14 +24,12 @@ data CLI = CLI {
     cliCoreNodeId   :: CoreNodeId,
     cliNumCoreNodes :: NumCoreNodes,
     cliProtocol     :: Protocol,
-    cliGenesis      :: Last PartialGenesis,
-    cliKeyMaterial  :: Last PartialStaticKeyMaterial
```

Comment:
You should also remove `Partialgenesis` and `PartailStaticKeyMaterial` from the import list; I should not pass the CI. If this did, file a ticket to update the nix part to run with `-Werror`; `-Wall` is already enabled in the `cabal` file.